siliconcompiler/toolscripts/_prereqs.sh

```
@@ -326,30 +346,55 @@ sc_remove_prereqs() {
+    # pattern, so "libllvm17*" matches the family without naming every member.
+    # shellcheck disable=SC2086
+    for _sc_bo_pat in $_sc_keep_pkgs; do
+        case "$1" in
+            $_sc_bo_pat) return 1 ;;
+        esac
+    done
+
+    # shellcheck disable=SC2086
+    for _sc_bo_pat in $_sc_drop_pkgs; do
+        case "$1" in
+            $_sc_bo_pat) return 0 ;;
```

Shellcheck (0.11.0)

[warning] Quote expansions in case patterns to match literally rather than as a glob.

(SC2254)